test/parallel/test-stream2-read-sync-stack.js

```diff
@@ -25,9 +25,6 @@ const Readable = require('stream').Readable;
 const r = new Readable();
 const N = 256 * 1024;
 
-// Go ahead and allow the pathological case for this test.
```

Comment:
Please keep (maybe move to L24 as per https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure)

Reply:
It no longer allows the pathological use case though since maxTickDepth was removed so the comment is incorrect (and has been since 0.12). Is there alternative phrasing you'd prefer?